### PR TITLE
📊 Migrate grapher charts to Data Pages (Pablo)

### PR DESCRIPTION
**Pull request by Marigold**
I've taken all Pablo's grapher charts from our [data pages candidate list](https://docs.google.com/spreadsheets/d/1SyyRR_iaokoc-tkDVQ24F4gW7cf47mLRdEyr8ZSBYo0/edit#gid=0) and ran `etl-metadata-migrate` to pre-generate YAML files. This should hopefully ease the migration. Generated YAML files contain all metadata information we have about indicator (including everything from garden step). This means we might end up with duplicate information in garden & grapher YAML files. It's up to you whether you decide to refactor it and move everything to `garden` channel. I don't think it's necessary now (maybe with next update).

## Instructions

Work on this branch and commit back to it. For all data pages from the list:

1. Look up their YAML file from this PR
2. Fill in all TBD fields (update or remove), remove `description` field
3. Push new commit which will trigger **ETL rebuild of staging server**
4. Wait until ETL is rebuilt
5. Look up relevant indicator for a chart on [staging](http://staging-site-data-pages-pablor/admin/variables) and open its data page

Alternatively, you can run it locally with
```
STAGING=data-pages-pablor etl ... --grapher
```

## Additional notes

### CODE REVIEW

**Comment by pabloarosado** [comment]
For some reason, data pages are not rendered for the modified indicators (e.g. [this one](http://staging-site-data-pages-pablor/grapher/access-to-clean-fuels-and-technologies-for-cooking)). However, on my local grapher, I can see them.

File: etl/steps/data/grapher/faostat/2023-06-12/faostat_qcl.meta.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+          Yield: Harvested production per unit of harvested area for crop products. In most of the cases yield data are not recorded but obtained by dividing the production data by the data on area harvested. Data on yields of permanent crops are not as reliable as those for temporary crops either because most of the area information may correspond to planted area, as for grapes, or because of the scarcity and unreliability of the area figures reported by the countries, as for example for cocoa and coffee. Source: FAO Statistics Division.
+        presentation:
+          topic_tags:
+            - Crop Yields
+          title_public: Cereal yields
+          attribution_short: FAO
+          grapher_config:
+            title: Cereal yields
+            subtitle: |-
```
Comment (veronikasamborska1994) [inline etl/steps/data/grapher/faostat/2023-06-12/faostat_qcl.meta.yml:58]: you have both `subtitle` and `description_short` but could just have one of these,  no?

File: etl/steps/data/grapher/worldbank_wdi/2023-05-29/wdi.meta.yml [hunk 2]
```diff
@@ -294,3 +294,71 @@ tables:
...
+
+          Statistical concept and methodology: Data on undernourishment are from the Food and Agriculture Organization (FAO) of the United Nations and measure food deprivation based on average food available for human consumption per person, the level of inequality in access to food, and the minimum calories required for an average person.
+        presentation:
+          title_public: Share of the population that is undernourished
+          attribution: Food and Agriculture Organization of the United Nations (via World Bank)
+          attribution_short: World Bank
+          topic_tags:
+            - Hunger & Undernourishment
+        processing_level: minor
```
Comment (veronikasamborska1994) [inline etl/steps/data/grapher/worldbank_wdi/2023-05-29/wdi.meta.yml:364]: can include number of decimal places displayed as 1

**Review by veronikasamborska1994 — APPROVED** [review]
LGTM

File: etl/steps/data/grapher/faostat/2023-06-12/faostat_qcl.meta.yml (hunk 1 above)
Comment (pabloarosado) [reply etl/steps/data/grapher/faostat/2023-06-12/faostat_qcl.meta.yml:58]: Yes, but they are actually different. Half of the subtitle goes to `description_short` (mentioning the unit of measurement) and part to the `description_key` (mentioning the definition of cereals).

Comment (veronikasamborska1994) [reply etl/steps/data/grapher/faostat/2023-06-12/faostat_qcl.meta.yml:58]: yep, it's all good. I just wasn't sure if there is any added value in having both.

**Pull request merged**